A&S/isUnique.js

```diff
@@ -0,0 +1,18 @@
+
+function isUnique(str) {
+    var strSet = new Set(str);
```

Comment:
what if this line is not available as a utility to be used? How else can we implement the same function?